multiqc/modules/picard/CrosscheckFingerprints.py

```diff
@@ -0,0 +1,236 @@
+    lod_threshold_regex = r"LOD_THRESHOLD=(\S+)"
+
+    tumor_awareness = None
+    lod_threshold = None
+
+    tumor_awareness_match = re.search(tumor_awareness_regex, line)
+    if tumor_awareness_match is not None:
+        tumor_awareness = strtobool(tumor_awareness_match.group(1))
+
+    lod_threshold_match = re.search(lod_threshold_regex, line)
+    if lod_threshold_match is not None:
+        lod_threshold = float(lod_threshold_match.group(1))
+
+    return (tumor_awareness, lod_threshold)
+
+
+def _get_name(row):
+    """ Make a name for the row based on the Samples and Groups present """
+    if (
+        row["LEFT_GROUP_VALUE"] == row["LEFT_SAMPLE"]
+        and row["RIGHT_GROUP_VALUE"] == row["RIGHT_SAMPLE"]
+    ):
+        return "{} - {}".format(row["LEFT_SAMPLE"], row["RIGHT_SAMPLE"])
+    else:
+        return "{}/{} - {}/{}".format(
+            row["LEFT_SAMPLE"],
+            row["LEFT_GROUP_VALUE"],
+            row["RIGHT_SAMPLE"],
+            row["RIGHT_GROUP_VALUE"],
+        )
+
+
+def _get_table_headers(data):
+    """ Create the headers config """
+
+    picard_config = getattr(config, "picard_config", {})
+    crosscheckfingerprints_table_cols = picard_config.get("CrosscheckFingerprints_table_cols")
+    crosscheckfingerprints_table_cols_hidden = picard_config.get(
+        "CrosscheckFingerprints_table_cols_hidden"
+    )
+
+    if not crosscheckfingerprints_table_cols:
+        crosscheckfingerprints_table_cols = [
+            "RESULT",
+            "DATA_TYPE",
+            "LOD_THRESHOLD",
+            "LOD_SCORE",
+        ]
+    if not crosscheckfingerprints_table_cols_hidden:
+        crosscheckfingerprints_table_cols_hidden = [
+            "LEFT_GROUP_VALUE",
+            "RIGHT_GROUP_VALUE",
+            "LEFT_RUN_BARCODE",
+            "LEFT_LANE",
+            "LEFT_MOLECULAR_BARCODE_SEQUENCE",
+            "LEFT_LIBRARY",
+            "LEFT_SAMPLE",
+            "LEFT_FILE",
+            "RIGHT_RUN_BARCODE",
+            "RIGHT_LANE",
+            "RIGHT_MOLECULAR_BARCODE_SEQUENCE",
+            "RIGHT_LIBRARY",
+            "RIGHT_SAMPLE",
+            "RIGHT_FILE",
+        ]
+
+        # Add the Tumor/Normal LOD scores if any pair had the tumor_awareness flag set
+        if any(row["TUMOR_AWARENESS"] for row in data.values()):
+            crosscheckfingerprints_table_cols += [
+                "LOD_SCORE_TUMOR_NORMAL",
+                "LOD_SCORE_NORMAL_TUMOR",
+            ]
+        else:
+            crosscheckfingerprints_table_cols_hidden += [
+                "LOD_SCORE_TUMOR_NORMAL",
+                "LOD_SCORE_NORMAL_TUMOR",
+            ]
+
+        headers = OrderedDict()
+        for h in FIELD_DESCRIPTIONS:
+            # Skip anything not set to visible
+            if h not in crosscheckfingerprints_table_cols:
+                continue
+
+            # Set up the configuration for the column
+            h_title = h.replace("_", " ").strip().lower().capitalize()
```

Comment:
answer is here for if the configuration key can be lowercase.  Can you update the doc?